guides/common/modules/proc_managing-ssh-keys-for-a-user-by-using-cli.adoc

```diff
@@ -1,35 +1,16 @@
 :_mod-docs-content-type: PROCEDURE
 
-[id="Managing_SSH_Keys_for_a_User_{context}"]
-= Managing SSH keys for a user
+[id="managing-ssh-keys-for-a-user-by-using-cli"]
+= Managing SSH keys for a user by using Hammer CLI
 
-Use this procedure to add or remove SSH keys for a user.
-To use the CLI instead of the {ProjectWebUI}, see the xref:cli-managing-ssh-keys-for-a-user_{context}[].
+[role="_abstract"]
+Use this procedure to add or remove SSH keys for a user by using Hammer CLI.
 
 .Prerequisites
 * Ensure that you are logged in to the {ProjectWebUI} as an Admin user of {ProjectName} or a user with the _create_ssh_key_ permission enabled for adding SSH key and _destroy_ssh_key_ permission for removing a key.
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Administer* > *Users*.
-. From the *Username* column, click on the username of the required user.
-. Click on the *SSH Keys* tab.
-+
-* To Add SSH key
-... Prepare the content of the public SSH key in a clipboard.
-... Click *Add SSH Key*.
-... In the *Key* field, paste the public SSH key content from the clipboard.
-... In the *Name* field, enter a name for the SSH key.
-... Click *Submit*.
-+
-* To Remove SSH key
-... Click *Delete* on the row of the SSH key to be deleted.
-... Click *OK* in the confirmation prompt.
-
-[id="cli-managing-ssh-keys-for-a-user_{context}"]
-.CLI procedure
-
-To add an SSH key to a user, you must specify either the path to the public SSH key file, or the content of the public SSH key copied to the clipboard.
-
+. To add an SSH key to a user, you must specify either the path to the public SSH key file, or the content of the public SSH key copied to the clipboard:
```

Comment:
I am unsure if this should be an ordered list. "Managing" includes many tasks; they're all on the same level for me. So:

```suggestion
* To add an SSH key to a user, you must specify either the path to the public SSH key file, or the content of the public SSH key copied to the clipboard:
```

the same applies to the other steps below. What do you think?